### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by Zi0nQ** [tsunamods-codes/7th-Heaven#139] [Q&A] [answered]
I didn't wanted to open a new topic, but my last one is closed. Can someone give the link to discord where I can ask for issues? Or the support channels? I'm trying to find it, but I don't know where. And if it's the Tsunamods dicrod, I don't know how to unlock it.

**Comment by JuakinBM** [comment]
Discord link it's not working on their website or in their twitter...

**Comment by julianxhokaxhiu** [comment, marked as answer]
The link on the website is valid, just click on `Tsunamods Discord` here https://7thheaven.rocks/#download7h and then click on the `Final Fantasy 7` button in `UNLOCK-DISCORD`.
